packages/expo-video/android/src/main/java/expo/modules/video/VideoPlayer.kt

```diff
@@ -137,6 +139,7 @@ class VideoPlayer(context: Context, appContext: AppContext, source: VideoSource?
       }
       if (playbackState == Player.STATE_READY) {
```

Comment:
shouldn't this have some sort of undefined state while the media item is loading? Are we making it explicit that you should not check this while the state is not ready?

Reply:
@aleqsio I don't know if adding a special state while the media is loading isn't a bit overkill. I think we can just set it to false.